src/coreComponents/mesh/FaceManager.cpp

```diff
@@ -604,7 +604,7 @@ void FaceManager::computeGeometry( NodeManager const & nodeManager )
 
     // This needs to be done somewhere else, also we probably shouldn't be orienting the normals like this.
     // Set normal orientation according to a global criterion
-    computationalGeometry::FixNormalOrientation_3D( m_faceNormal[ faceID ] );
+//    computationalGeometry::FixNormalOrientation_3D( m_faceNormal[ faceID ] );
```

Comment:
@andrea-franceschini This breaks the assumed convention of external faces having outward pointing normals. Is this still required?

Reply:
You are right. This was used to have a consistent orientation for a fracture, that, being an internal surface, does not have an outward direction. But (I assume) we can safely comment it out. @castelletto1 do you think there are specific constraints to keep it?